src/containers/sidebars/aoi-sidebar/species-card/index.js

```diff
@@ -137,6 +141,21 @@ function SpeciesCardContainer(props) {
     );
   };
 
+  useEffect(() => {
+    EsriFeatureService.getFeatures({
+      url: LAYERS_URLS[AOI_SPS_TABLE],
+      whereClause: `GID_0 = '${contextualData.iso}'`,
```

Comment:
This for now will only work on countries but we will have to make it work for all AOIs. Maybe some comment about this will be fine